packages/payment/src/shared/index.ts

```diff
@@ -136,6 +136,8 @@ export interface CreateApplePayOption {
   countryCode: string;
   currency: string;
   requiredShippingContactFields?: ('postalAddress' | 'phoneNumber' | 'emailAddress' | 'name')[];
+  allowedCountries?: string[];
+  allowedCountriesError?: string;
```

Comment:
The variable name `allowedCountriesError` makes it hard to predict whether it is an Error Object, Error Code, or something else,

- allowedCountriesErrorDescription
- allowedCountriesErrorMessage

is better to use either of the following. (The same goes for code in Swift)

Reply:
Hi @rdlabo, I did the change.

Thank you for including this change, in my company this is something that helps us a lot since this way we don't have to work with a fork and make our changes.

Delighted to be able to contribute and thanks to your team for maintaining this plugin.

Reply:
Thanks for the quick change. It's perfect!